specifyweb/frontend/js_src/lib/components/FormPlugins/PartialDateUi.tsx

```diff
@@ -49,18 +49,20 @@ export function PartialDateUi({
     React.useContext(ReadOnlyContext) || resource === undefined;
 
   return (
-    <div className="flex w-full gap-1">
+    <div className="grid w-full grid-cols-[fit-content(40%)_1fr] gap-1">
```

Comment:
if we find that `fit-content(40%) 1fr` is too complicated or doesn't work fine across browsers, setting this to `2fr 3fr` works almost just as well